.readme-partials.yml

```diff
@@ -0,0 +1,7 @@
+body: |-
+        ### Troubleshooting
+        #### Emulator returning `DEADLINE_EXCEEDED`, `java.lang.OutOfMemoryError`
+        *Reference Issue: https://github.com/googleapis/nodejs-datastore/issues/95*
```

Comment:
it might be nice to make this a link:

```
*Reference Issue: [#95](https://github.com/googleapis/nodejs-datastore/issues/95)*
```

Reply:
Do you mean make the text “Reference Issue” a link rather than the issue number? This is a weird consistency micro-concern of an edge case where we have multiple reference links.

Reply:
Oh man, did my eyes trick me on this. Sorry, I'll make that change. What a goofball.

Reply:
I wasn't sure if GitHub linkifies links in READMEs, so was just thinking it's nice to have something clickable, we can confirm either way when we generate the README.

Reply:
Yeah, sorry, I thought you pasted the line the way I had it already, but you were actually writing out your suggestion. It was such a good suggestion, that I assumed I had already done it! 🤣